### ISSUE TITLE
Accessibility issues with ion-toast component

### ISSUE THREAD

**Issue by mastergogo** [ionic-team/ionic-framework#22333]
It has been observed that, whenever ion-toast is triggered by a tooltip or by a button, when toast appears screen readers such as Android's talk back and iOS's Voice over, they aren't able to announce the toast content. it can be resolved by two ways, toast should either have a aria live-region in it so that the moment it appears screen reader can announce the content, or the other way would be to move focus to the toast container so screen reader user can access the toast content.

In my ionic app, we are trying to move the focus to programmatically to toast so screen readers can access the toast content, however it works only for android, it does not work for iOS for some reason, therefore the common solution we are trying in aria-live region. could this implemented within the toast itself so we get it out of the box?

Issue can be reproduced with this [link](https://ionic-docs-demo.herokuapp.com/component/toast):

1) open the above link and navigate using talk back or voice over to the button
2) double tap on the button and toast will appear however, screen reader user is not made aware of that

Let me know if more info is needed.

Thanks,
Amit

**Comment by liamdebeasi** [comment]
Thanks for the issue. This has been resolved via https://github.com/ionic-team/ionic-framework/pull/24937, and a fix will be available in an upcoming release of Ionic Framework.
